src/components/SeedBank/SeedBankSubLocations.tsx

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from 'react';
+import { APP_PATHS } from 'src/constants';
+import { ActiveStatuses } from 'src/types/Accession';
+import { PartialSubLocation } from 'src/types/Facility';
+import SubLocations from 'src/components/common/SubLocations';
+
+export type SeedBankSubLocationsProps = {
+  seedBankId?: number;
+  onEdit?: (subLocations: PartialSubLocation[]) => void;
+};
+
+export default function SeedBankSubLocations({ seedBankId, onEdit }: SeedBankSubLocationsProps): JSX.Element {
```

Comment:
renamed SubLocations to SeedBankSubLocations and moved the original content over to common subdirectory.. this component here is now a wrapper around SubLocations